app/services/promo_registrar.rb

```diff
@@ -15,7 +15,11 @@ def perform
         referral_code = result[:referral_code]
         should_update_promo_server = result[:should_update_promo_server]
         if referral_code.present?
-          promo_registration = PromoRegistration.new(channel_id: channel.id, promo_id: @promo_id, kind: "channel", referral_code: referral_code)
+          promo_registration = PromoRegistration.new(channel_id: channel.id,
+                                                     promo_id: @promo_id,
+                                                     kind: "channel",
```

Comment:
Can we use the constant here?
```suggestion
                                                     kind: PromoRegistration::CHANNEL,
```

Reply:
gonna try this commit suggestion feature